.travis.yml

```diff
@@ -1,76 +1,176 @@
+        - echo "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURvakNDQW9xZ0F3SUJBZ0lVVHFDWVpaVXVnQUQyeXZsL2x0cGtFWkVYY2hjd0RRWUpLb1pJaHZjTkFRRUwKQlFBd2ZqRUxNQWtHQTFVRUJoTUNSa2t4RURBT0JnTlZCQWdNQjBacGJteGhibVF4RGpBTUJnTlZCQWNNQlVWegpjRzl2TVF3d0NnWURWUVFLREFORFUwTXhFREFPQmdOVkJBc01CMU41YzBSbGRuTXhFVEFQQmdOVkJBTU1DRXh2ClkyRnNSVWRCTVJvd0dBWUpLb1pJaHZjTkFRa0JGZ3QwWlhOMFFHTnpZeTVtYVRBZUZ3MHhPREE1TVRRd056UTMKTWpGYUZ3MHlNVEEyTVRBd056UTNNakZhTUg0eEN6QUpCZ05WQkFZVEFrWkpNUkF3RGdZRFZRUUlEQWRHYVc1cwpZVzVrTVE0d0RBWURWUVFIREFWRmMzQnZiekVNTUFvR0ExVUVDZ3dEUTFORE1SQXdEZ1lEVlFRTERBZFRlWE5FClpYWnpNUkV3RHdZRFZRUUREQWhNYjJOaGJFVkhRVEVhTUJnR0NTcUdTSWIzRFFFSkFSWUxkR1Z6ZEVCamMyTXUKWm1rd2dnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUUM4bHVlaE1uN25Id2EvVUNqSgpHZGlpZXEyT0ZCZFF1Z0FnaDBaYjNsZ2JWMFAxTncxZmlJS2VoclFHSkUvZFgrOUpIZXoweVRYd0RlVEt3czBMCmpjYzN6UzdFbVJhaGxkd096YjlPWDVwNjdISHBaYmovT084NDF1cnZFQXkrUjgzRHpSaWFCaUFvbkZOSndIOUIKb2NrazBkcjlKa3RmN2Z4R3BGS2ZWalNaNlZ2OHROMi9KVXZBYnlsMjI5aEpCaXFzczRSMVpZa2hVSFExN0tDaQoxQXRzMXBIVHdDZjh1ZVlQVHoyUkl1VlQ1M2hVWVRTa3h3VmtBTU5QaGt6N1hMRUxYU2VyRHpqUmZKUG5hZnpzCkZGOG9TNjVFWnZRQVpPWVN5c1dsMklzK05HZ1R6bk4yMXcxUGhuaUkxdzJjRmZXb09mVE5LK21tKzhhMTkvTzQKcmQrckFnTUJBQUdqR0RBV01CUUdBMVVkRVFRTk1BdUNDV3h2WTJGc2FHOXpkREFOQmdrcWhraUc5dzBCQVFzRgpBQU9DQVFFQVJOSkpQemg2WGdjUVNCWFU3M2JZVkt6UmtSeUpRVGtNOHc4Z01COE1LNi9XYk9LcjJhZnZYcmQrClpoZWpVUUVxS3FONmNSenpJZjdsZ1N3UDdsaXU3WkJTNHozVG16MXYydk9CN2NVS25QV3M1cW1WM1RPQjlobVYKUFNEUTJKQ3BRRkV1N0JRcmI1UVZTdUNwQ2FzSXNXYzVZQlVaT0V6K3JFQ1ZkZkY3Z29zSnM5RGhpc1I5WE5rbwowYk83cERRSmkycmJTbEtoNS9LNlk1KzEwQ1N1K1lzOWI4YUZ2b3daTFpRYVJqRTIyNUhuVzFGSHpzZUNuOXJwCjJPbHRLRlJ4S3JzZDV2VnhlOTFXaHdNQjFscmc3UkdZa0ZYdnladW9PZ09NSVlnRFNxNDhFZ1VrNTBHSW80SXYKTHI4VmRTYkN1OG1oZFo2a3pid0F6cG1lcXNXa0VRPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=" | base64 -d > ega-charts/localega/config/ssl.cert
+        # install LocalEGA
+        - helm install --name localega ega-charts/localega --set persistence.enabled="false",config.broker_username="$MQUSER",config.cega_endpoint="$CEGA_ENDPOINT",config.cega_mq_host="cega-mq",config.cega_username="lega",config.cega_vhost="lega",config.data_storage_url="http://minio:9000",config.postgres_user="lega",secrets.cega_creds="$CEGA_CREDS",secrets.cega_mq_pass="$CEGA_MQ_PASS",secrets.keys_password="$KEYS_PASSWORD",secrets.lega_password="$LEGA_PASSWORD",secrets.mq_password="$MQ_PASSWORD",secrets.postgres_password="$POSTGRES_PASWORD",secrets.s3_access="$MINIO_ACCESS",secrets.s3_secret="$MINIO_SECRET"
+        - sleep 180
```

Comment:
we will need a similar preflight mechanism, that will check when the services are up - some sort of health check, although not all services provide one.

Reply:
If you look at the lines below that is what they do. In the sense that we make sure that everything is up and running.

Reply:
true, then we don't need to sleep here ?